[38;21m2025-10-16 04:41:08,464 :: DEBUG :: icloudpy.base :: base.py :: 240 :: Using session file /tmp/icloudpy/runner/quentintarantinohotmailfr.session[0m
[38;5;39m2025-10-16 04:41:08,464 :: INFO :: icloudpy.base :: base.py :: 247 :: Session file does not exist[0m
[38;21m2025-10-16 04:41:08,465 :: DEBUG :: icloudpy.base :: base.py :: 313 :: Authenticating as quentintarantino@hotmail.fr[0m
[38;21m2025-10-16 04:41:08,473 :: DEBUG :: icloudpy.base :: base.py :: 417 :: Authentication completed successfully[0m
[38;21m2025-10-16 04:41:08,475 :: DEBUG :: root :: drive_file_existence.py :: 46 :: No changes detected. Skipping the file /home/runner/work/icloud-docker/icloud-docker/tests/temp/icloud/drive/Scanned document 1.pdf ...[0m
